xbmc/pvr/providers/PVRProviders.h

```diff
@@ -0,0 +1,122 @@
+/*
+ *  Copyright (C) 2012-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "threads/Thread.h"
```

Comment:
Is this include needed? Or is the include for `CCriticalSection` sufficient?

Reply:
Yup, CCriticalSection is good enough.